service_files/xyz.openbmc_project.DataSync.BMCData.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Redundant BMC Data Synchronization
+Wants=xyz.openbmc_project.State.BMC.Redundancy.service
+After=xyz.openbmc_project.State.BMC.Redundancy.service
+
+[Service]
+ExecStart=/usr/libexec/phosphor-data-sync/phosphor-rbmc-data-sync-mgr
+Restart=always
+Type=dbus
+BusName=xyz.openbmc_project.DataSync.BMCData
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I think we discussed you'll remove the multi-user.target dep for now?

Reply:
Yes, for now - Just link with active and passive targets

Reply:
Done